python/paddle/v2/framework/tests/test_maxout_op.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+
+def maxout_forward_naive_2sweetsky(input, groups, num_channels):
```

Comment:
If this function is not used, please to remove it.

Reply:
done